controllers/numaresourcesoperator_controller.go

```diff
@@ -210,6 +212,15 @@ func (r *NUMAResourcesOperatorReconciler) syncNUMAResourcesOperatorResources(ctx
 	return daemonSetsNName, nil
 }
 
+func (r *NUMAResourcesOperatorReconciler) getExporterImage(imgSpec string) string {
+	imageSpec := imgSpec
+	if imageSpec == "" {
+		imageSpec = r.ImageSpec
+	}
+	klog.InfoS("Using Exporter", "image", imageSpec)
```

Comment:
The log message isn't completely clear.
I would let the user know if we override the default image using the API filed or not

Reply:
ok, I'll revisit the log line.